test/release/examples/primers/modules.chpl

```diff
@@ -238,6 +319,11 @@ module MainModule {
       } // Will output modToUse.foo (which is '12')
     }
 
+    /* The only way to ``import`` multiple modules in a single statement is to
+       name a shared parent module.  We will talk about this more in
+       :ref:`Primer_Nested_Modules`.
```

Comment:
I always confuse myself... is this a current limitation or by design? If it is the former, does it worth mentioning here that it could be supported in the future?

Reply:
It's a current limitation.  I think in the past we've shied away from "future-looking" language in the primers?  We'll definitely update this when we implement it

Reply:
I wasn't aware of that policy but it makes sense